fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+        
+        
+      
+      parameters:
+        - name: id
+          in: path
+          required: true
+          type: integer
+      responses:
+        '201':
+          description: Checkout object created or retrieved.
+          schema:
+            $ref: '#/definitions/checkout'
+            
+        '400':
+          description: Bad Request
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'
+                  
+            
+            
+      
+      tags:
+        - Create
+        - To Be Implemented
+        
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X POST -H "Content-Type: application/json" https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin/draft_orders/#{id}/checkout.json
+            
+        - lang: Ruby
+          source: |
+            require 'shopify_api'
+
+            shop_url = "https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin"
+            ShopifyAPI::Base.site = shop_url
+            draft_order = ShopifyAPI::DraftOrder.find(#{id})
+            draft_order.checkout    
+            
+definitions:
+  DraftOrder:
+    type: object
+    properties:
+      id:
+        type: string
+        description: The `id` of the draft order.
+        readOnly: true
+      order_id:
+        type: string
+        description: The `id` of the order associated to the draft order, once created.
+        readOnly: true
+      name:
+        type: string
+        description: Name of the draft order.
+        readOnly: true
+        
+      customer_id:
+        type: string
+        description: Customer `id` used to associate the customer to the draft order.  Set to null to remove the customer from the draft order.
+        
+      customer:
+        $ref: '#/definitions/customer'
+        
+      shipping_address:
+        $ref: '#/definitions/shipping_address'
+      billing_address:
+        $ref: '#/definitions/billing_address'
+      note:
+        type: string
+        description: The text of an optional note that a merchant can attach to the draft order.
+        readOnly: true
+      email:
+        type: string
+        description: The email address used for sending notifications.  For example, the customer email address, if a customer is associated to the draft order.
+      currency:
+        type: string
+        description: The three letter code for the currency to be used for the payment.
+        readOnly: true
+      invoice_sent_at:
+        type: string
+        description: Date when the invoice was emailed to the customer.
+        format: date-time
+        readOnly: true
+    
+      invoice_url:
+        type: string
+        description: Url sent to the customer to complete web checkout.  When using the `send_invoice.json` call, the url is emailed to the customer.
```

Comment:
* begin line with "The..."
* let's go with "URL" instead of "Url"
* "to complete web checkout" -> "to complete their web checkout"